Library/Formula/duck.rb

```diff
@@ -0,0 +1,29 @@
+class Duck < Formula
+  homepage "https://duck.sh/"
+  url "https://dist.duck.sh/duck-src-4.6.2.16342.tar.gz"
+  version "4.6.2.16342"
```

Comment:
Don't need version now it's autodetected from the URL: http://bot.brew.sh/job/Homebrew%20Pull%20Requests/19549/version=yosemite/testReport/junit/brew-test-bot/yosemite/audit_duck___strict/